utils/common/src/main/java/org/apache/brooklyn/util/net/ReachableSocketFinder.java

```diff
@@ -91,7 +91,7 @@ public HostAndPort findOpenSocketOnNode(final Collection<? extends HostAndPort>
                 .backoffTo(Duration.FIVE_SECONDS)
                 .until(new Callable<Boolean>() {
                         public Boolean call() {
-                            Optional<HostAndPort> reachableSocket = tryReachable(sockets, Duration.seconds(2));
+                            Optional<HostAndPort> reachableSocket = tryReachable(sockets, Duration.FIVE_SECONDS);
```

Comment:
I think this should be set in the predicate or configurable. It's implementation dependant could vary.